PWGLF/Tasks/Resonances/lambda1520SpherocityAnalysis.cxx

```diff
@@ -351,12 +351,61 @@ struct lambdaAnalysis {
     return true;
   }
 
+  template <typename T>
+  void fillQAHistos(T const& track)
+  {
+
+    // get total momentum
+    float p = TMath::Sqrt(track.px() * track.px() + track.py() * track.py() + track.pz() * track.pz());
```

Comment:
```suggestion
    float p = track.p();
```

Reply:
I think it is not available in resodaughter table in LFResonanceTables.h so I can't use it.

Reply:
Okay I will do that. Thank you.